src/test/java/org/assertj/core/osgi/soft/CustomSoftAssertionTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.osgi.soft;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.assertj.core.api.AbstractListAssert;
+import org.assertj.core.api.AbstractMapAssert;
+import org.assertj.core.api.AbstractSoftAssertions;
+import org.assertj.core.api.ObjectAssert;
+import org.assertj.core.api.ProxyableListAssert;
+import org.junit.jupiter.api.Test;
+
+public class CustomSoftAssertionTest {
+
+  @Test
+  void custom_soft_assertions_success() {
+    Map<String, String> map = new HashMap<>();
+    map.put("key1", "value1");
+    map.put("key2", "value2");
+
+    TestSoftAssertions softly = new TestSoftAssertions();
+    softly.assertThat(map).containsKeys("key1", "key1").containsValues("value1", "value2");
```

Comment:
typo: `"key1", "key2"`

Reply:
Oops! I will fix.

Reply:
Maybe this is a consideration for an enhancement- containsKeys() could throw an IllegalArgumentException if you pass in a duplicate? Would help catch accidental errors ingress this.